### PR TITLE
Renault Zoe Ph1

### PR DESCRIPTION
**Pull request by Dimitrie78**
add Cabin Pre-heat/cool Control

### CODE REVIEW

File: vehicle/OVMS.V3/components/vehicle_renaultzoe/docs/index.rst [hunk 1]
```diff
@@ -37,4 +37,11 @@ Others
 After selecting the Renault Zoe/Kangoo vehicle module, the corresponding settings have to be made in the web interface via the "Renault Zoe/Kangoo" menu under "Features":
 
 .. image:: Features.png
-    :width: 480px
\ No newline at end of file
+    :width: 480px
+
+
+----------
```
Comment (dexterbg) [inline vehicle/OVMS.V3/components/vehicle_renaultzoe/docs/index.rst:43]: Both headline indicator lengths needs to match the headline length

File: vehicle/OVMS.V3/components/vehicle_renaultzoe/docs/index.rst [hunk 2]
```diff
@@ -37,4 +37,11 @@ Others
...
+Using Cabin Pre-heat/cool
+----------
+
+Only 5 Minutes Booster are Implementet white Climatecontrol on or Homelink 1
```
Comment (dexterbg) [inline vehicle/OVMS.V3/components/vehicle_renaultzoe/docs/index.rst:47]: This is hardly readable or understandable… please clarify and fix spelling. If I understand you correctly, this also contradicts the much broader claim you added to "changes.txt".

File: vehicle/OVMS.V3/components/vehicle_renaultzoe/src/rz_web.cpp [hunk 3]
```diff
@@ -131,6 +134,9 @@ void OvmsVehicleRenaultZoe::WebCfgFeatures(PageEntry_t& p, PageContext_t& c)
   c.input_select_option("Kangoo",    "1", vehicle_type == "1");
   c.input_select_end();
   
+  c.print("<hr>");
+  c.input_checkbox("use EGPIOs for Homelink", "use_gpios", use_gpios,
+    "<p>Disabeld are Default use for Homelink 1 = Climatecontrol for IOS users. When enable some EGPIOs are switched. (BINKINO thinks)</p>");
```
Comment (dexterbg) [inline vehicle/OVMS.V3/components/vehicle_renaultzoe/src/rz_web.cpp:139]: This is an even worse explanation. No normal user will understand this, and software should generally not describe it's functionality by means of "some" and "thinks". It should exactly define what it does.

**Review by dexterbg — COMMENTED** [review]
Dimitrie, some minor issues here regarding your documentation.

**Comment by Dimitrie78** [comment]
i updated the descriptions

**Pull request merged**